notNeededPackages.json

```diff
@@ -4475,6 +4475,10 @@
             "libraryName": "rehype-react",
             "asOfVersion": "6.0.0"
         },
+        "relay-config": {
+            "libraryName": "relay-compiler",
+            "asOfVersion": "13.0.0"
```

Comment:
Hm, this version doesn't actually exist; I'm not sure what affect this'll have. Maybe the package should just be deleted? I'm not sure what precedent there is for effectively deprecated packages, though `relay-compiler` doesn't list itself as deprecated on npm.

(Maybe @sandersn or @andrewbranch know more.)

Reply:
the types for relay-config is no longer a thing. You're right. I just tried to follow the guides, if simply deleting the types, I can do this 🚀

Reply:
To be clear, the package still exists as v12 on npm, it's just that the package is deprecated, right? I'd think you'd want to deprecate it upstream first.

Reply:
the Facebook maintainers just deleted it from their GitHub repo. I'll reach out and ask if they can deprecate 😅

Reply:
Oh, well, that certainly doesn't help. I guess it can just be deleted altogether, then.

Reply:
I've deleted the changes to the config file.